### PR TITLE
[ja] update controllers and core-libraries.

### PR DESCRIPTION
**Pull request by okinaka**

### CODE REVIEW

File: ja/controllers/request-response.rst [hunk 1]
```diff
@@ -760,14 +788,25 @@ Not-Modified レスポンスの送信
 .. versionadded:: 3.2
     ``CorsBuilder`` は 3.2 で追加されました。
 
-レスポンスの送信
-----------------
+不変レスポンスに伴うよくある失敗
+========================================
+
+CakePHP 3.4.0 以降、レスポンスオブジェクトはレスポンスを不変オブジェクトとして扱う
+いくつかのメソッドを提供しています。不変オブジェクトは、偶発的な副作用の追跡を困難にし、
```
Comment (chinpei215) [inline ja/controllers/request-response.rst]: > Immutable objects help **prevent** difficult to track accidental side-effects,

Probably, the above 'prevent' would be 'bare infinitive'. (原型不定詞)

不変オブジェクトは、偶発的な副作用の追跡が困難になるのを**予防し**、

**Comment by chinpei215** [comment]
Also, please take a look at the following. The line 250 in the English version. `return isset($request->awesome);`, has been changed to `return $request->getParam('awesome');`. Also, the line 258 in the English version, `return $request->param('controller') === $name;` has been changed to `return $request->getParam('controller') === $name;`.

**Comment by okinaka** [comment]
@chinpei215 Thanks! I updated.

**Comment by chinpei215** [comment]
@okinaka Nice work!

**Pull request merged**